config.json

```diff
@@ -1,102 +1,100 @@
 {
-  "language": "Ceylon",
   "active": false,
   "exercises": [
     {
-      "uuid": "9ae32021-a7c3-4ddf-88b1-629b708b9cb9",
-      "slug": "leap",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 1,
+      "slug": "leap",
       "topics": [
-        "Integers"
-      ]
+        "integers"
+      ],
+      "unlocked_by": null,
```

Comment:
shouldn't it be the case that unlocked_by and core are next to each other? They seem too closely related to separate in this manner.